### PR TITLE
fix: update config.toml values

### PR DESCRIPTION
**Pull request by KwilLuke**
I noticed a handful of config values added in v0.8 are missing, so I am adding them here.

I also updated notes on default values per the value set here: https://github.com/kwilteam/kwil-db/blob/99ea751131a70fe027d6f0515ea1bb188e0bdd14/cmd/kwild/config/config.go#L556

### CODE REVIEW

File: docs/node/daemon/config/settings.mdx [hunk 1]
```diff
@@ -12,46 +12,58 @@ This section provides a detailed overview of the operational parameters availabl
...
-   - `pg_db_pass`: PostgreSQL database pass (may be omitted for some pg_hba.conf configurations).
+   - `pg_db_pass`: PostgreSQL database pass (may be omitted for some pg_hba.conf configurations). No default.
 
    - `pg_db_name`: PostgreSQL database name (override database name). The default is `kwild`.
 
    - `private_key_path`: This variable specifies the path for node's private key. If left unspecified, it defaults to `./private_key` relative to the root directory.
 
+   - `profile_mode`: This variable sets the kwild profiling module. Options are `http`, `cpu`, `mem`, `mutex`, and `block`. No default.
+
+   - `profile_file`: The kwild profile output file path. Not enabled by default.
```
Comment (jchappelow) [inline docs/node/daemon/config/settings.mdx:48]: Could add that this setting does not apply for `http` profile mode.

File: docs/node/daemon/config/settings.mdx [hunk 2]
```diff
@@ -12,46 +12,58 @@ This section provides a detailed overview of the operational parameters availabl
...
-   - `hostname`: This variable sets the Kwil JSON-RPC server's hostname.
+   - `hostname`: This variable sets the Kwil JSON-RPC server's hostname. No default.
 
-   - `genesis_state`: This variable specifies the path to the snapshot file to be used for the initial state of the node. If the node is configured to start with initial state and this variable is not set, the node will fail to start.
+   - `genesis_state`: This variable specifies the path to the snapshot file to be used for the initial state of the node. If the node is configured to start with initial state and this variable is not set, the node will fail to start. No default.
 
    ### [app.snapshots]
 
-      This section contains configurations for managing the snapshots of the kwild database.
+      This section contains configurations for managing the snapshots of the kwild database. Snapshot variables do not have a default value.
```
Comment (jchappelow) [inline docs/node/daemon/config/settings.mdx:66]: @charithabandi is it right that these don't have defaults?  I believe these are set:

```go
			Snapshots: SnapshotConfig{
				Enabled:         false,
				RecurringHeight: 14400, // 1 day at 6s block time
				MaxSnapshots:    3,
				SnapshotDir:     SnapshotDirName, // "snapshots"
			},
```

File: docs/node/daemon/config/settings.mdx [hunk 3]
```diff
@@ -90,49 +102,53 @@ This section contains configuration specific to the CometBFT-based blockchain.
...
    
       This section discusses the configurations for the blockchain's transaction mempool.
 
-      - `size`: This variable specifies the mempool's maximum transaction count.
+      - `size`: This variable specifies the mempool's maximum transaction count. Default is 5000.
+
+      - `max_tx_byte`: This variable specifies the maximum size of any one transaction in the mempool. Default is 4MB.
 
-      - `max_txs_bytes`: This variable specifies the limit on the total size of all transactions in the mempool.
+      - `max_txs_bytes`: This variable specifies the limit on the total size of all transactions in the mempool. Default is 512 MB.
```
Comment (jchappelow) [inline docs/node/daemon/config/settings.mdx:135]: To be pedantic, MiB.  `MaxTxsBytes: 1024 * 1024 * 512, // 512 MiB`  so not 512,000,000, but 536,870,912 or 2^29

File: docs/node/daemon/config/settings.mdx [hunk 4]
```diff
@@ -144,39 +160,48 @@ Here is an example of a `config.toml` file with complete configuration:
...
 # RootDir/
 #   |- config.toml   (app and chain configuration for running the kwild node)
-#   |- genesis.json  (genesis file for the network)
 #   |- private_key   (node's private key)
 #   |- abci/
-#   |   |- addrbook.json  (peer routable addresses for the kwild node)
-#   |   |- data/          (blockchain files: blockstore.db, state.db, etc)
+#   |   |- config/
+#   |   |   |- genesis.json   (genesis file for the network)
+#   |   |   |- addrbook.json  (peer routable addresses for the kwild node)
```
Comment (jchappelow) [inline docs/node/daemon/config/settings.mdx:174]: This change isn't quite correct.  The genesis.json that users care about in most cases is still in RootDir.  This abci/config folder contains active config in the cometbft format.  This README might help explain better:

```
$  cat ~/.kwild/abci/config/README 
This config folder is used to echo the in-memory CometBFT configuration
that is generated from kwild's config and genesis files. This folder and the files
within are useful for running certain debugging commands with the official cometbft
command line application (github.com/cometbft/cometbft/cmd/cometbft).

WARNING: These files are overwritten on kwild startup
```


Also, afaik, addrbook.json is still in abci not abci/config.

Comment (KwilLuke) [reply docs/node/daemon/config/settings.mdx:174]: Okay - FYI, this is copy and paste from the autogen config.toml with `kwild --autogen` on v0.8.1. We may want to update the comments in that doc.

cc @charithabandi @brennanjl

File: docs/node/daemon/config/settings.mdx (hunk 3 above)
Comment (KwilLuke) [reply docs/node/daemon/config/settings.mdx:135]: Thank you.

**Comment by KwilLuke** [comment]
Realized I had addressed this comment weeks ago but did not push. Feel free to approve if it looks good to you.

Also @charithabandi, please see the comment about updating the comments in the auto-generated config.toml: https://github.com/kwilteam/docs/pull/124#discussion_r1653600801

**Pull request merged**